portfolio/src/main/webapp/script.js

```diff
@@ -21,6 +21,7 @@ let fetchedMarkers = [];
 let markerLat;
 let markerLng;
 let reportsForMarkers = [];
+var autocomplete;
```

Comment:
let instead of var to prevent scope errors https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#:~:text=let%20allows%20you%20to%20declare,function%20regardless%20of%20block%20scope.

Reply:
Oh, that makes sense, thank you!